client/app/lib/components/form/fields/SelectField.jsx

```diff
@@ -82,6 +82,7 @@ const FormSelectField = (props) => {
           options.map((option) => (
             <Option
               key={option.value}
+              disabled={option.disabled ?? false}
```

Comment:
is there a possibility that `option.disabled` is not defined?

Reply:
Yes there is. Other files that use the SelectField previously have not defined option.disabled. This is left like this for backwards compatibility as the default value is false as well

Reply:
understand